Automattic/sensei
# Add .nvmrc file with the current Node LTS version

For people who use `nvm` might be useful to have [a hint right in the code base](https://github.com/nvm-sh/nvm#nvmrc).

### Changes proposed in this Pull Request

* Add `.nvmrc` file containing current Node LTS version. 

### Testing instructions

* [Install NVM](https://github.com/nvm-sh/nvm#install--update-script) if you don't have it.
* Go to your terminal and navigate to the project root.
* Run `nvm use` and follow instructions.
* Check your current Node version: `node -v`.

### Screenshot / Video
https://user-images.githubusercontent.com/329356/157509290-4b606ca1-34c1-4ba3-ba7d-a89e6e1855c3.mp4
